src/types/UserRef.ts

```diff
@@ -1,16 +1,19 @@
 export default interface UserRef {
     id: number;
+    display_name: string;
+    avatar_url: string;
     /**
      * @deprecated Please use `email` instead.
      * @see email
      */
     username: string;
     email: string;
-    display_name: string;
     first_name: string | null;
-    avatar_url: string;
-    /**
-     * Last time the user was active. ISO-8601 format.
-     */
+
+    is_terms_of_service_accepted: boolean;
+    sign_in_flow: 'google' | 'password' | 'sso';
+    use_case_answer: string;
```

Comment:
It's an enum in `prezly`: https://github.com/prezly/prezly/blob/master/apps/backend/resources/javascripts/types/api/UseCaseAnswer.ts

Reply:
You're right. But should we expose a list of possible values to (3rd party) Prezly API consumers?